InventoryConfigurableProductFrontendUi/view/frontend/web/js/configurable-variation-qty.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/**
+ * Configurable variation left qty.
+ */
+define([
+    'jquery',
+    'underscore',
+    'mage/url'
+], function ($, _, urlBuilder) {
+    'use strict';
+
+    return function (productSku, salesChannel, salesChannelCode) {
+        var selectorInfoStockSkuQty = '.availability.only',
+            selectorInfoStockSkuQtyValue = '.availability.only > strong',
+            productQtyInfoBlock = $(selectorInfoStockSkuQty),
+            productQtyInfo = $(selectorInfoStockSkuQtyValue);
+
+        if (!_.isUndefined(productSku) && productSku !== null) {
+            $.ajax({
+                url: urlBuilder.build('inventory_catalog/product/getQty/'),
```

Comment:
Why we need addiotional call to retrieve such information? Why we cannot just add it to the template while processing on the backend?

Reply:
@sidolov we added this additional call because of https://jira.corp.magento.com/browse/MC-17694 (it is  in acceptance criteria). 
Theoretically we could add this information in template, but we don't know if the product's qty isn't changed while the customer is watching the page. So I think that we need this call

Reply:
Thank you for the clarification. I got all answers from the ticket comments.